woocommerce/woocommerce-gateway-stripe
# Fix: Disable confirmation modal showing for all methods having 'Stripe' in name

### Description
Disable confirmation modal is supposed to show up only for the main "Stripe" method when user clicks on `disable`. But is it opening for any method which has `Stripe` in it's name

### How to reproduce
- Enable `_wcstripe_feature_upe_settings`
- Enable Stripe
- Enable UPE checkout 
- Go to `http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout`
- Click to disable the main `Stripe` method. A confirmation modal appears and it is expected.
- Click to enable or disable `Stripe Alipay` or `Stripe Multibanco`. The confirmation modal appears for them also. This is not expected.

![modal on wrong method](https://user-images.githubusercontent.com/33387139/136431308-e03de1ce-e939-4b0b-8be7-da6921c7b1eb.gif)

### Testing instructions
- Follow the steps mentioned in `How to reproduce`
- Clicking to enable or disable `Stripe Alipay` or `Stripe Multibanco` does not show the confirmation modal anymore.
